packages/proxy/src/program.ts

```diff
@@ -18,12 +19,27 @@ export async function program(opts: ProgramOptions) {
 
   const start = await create({ port: proxyPort });
 
+  const stopWatching = createFileWatcher(fullyQualifiedPath, run);
   // Wrap up getting config and starting
   async function run() {
     const config = await getProxyConfig(fullyQualifiedPath);
-    await start(config);
+    const server = await start(config);
+
+    process.on("SIGINT", async () => {
+      // unset the proxy
+      await unsetProxy();
```

Comment:
all of these will go uncaught, which is "fine" but we could probably offer some nice debugg-ability (future us can do this, or perhaps hashtag good first issue)

Reply:
added todo